server/controllers/stripe.js

```diff
@@ -45,29 +45,51 @@ exports.retrieveCustomer = async (req, res) => {
   res.send(customer);
 };
 
-// @route POST /stripe/payment/create
-// @desc creates new payment method
+// @route GET /stripe/customers/retrieveAll
+// @desc retrieve customer
 // @access Public
-exports.createPaymentMethod = async (req, res) => {
-  const paymentMethod = await stripe.paymentMethods.create({
-    type: "card",
-    card: {
-      number: "4242424242424242",
-      exp_month: 12,
-      exp_year: 2022,
-      cvc: "314",
-    },
-  });
-  res.send(paymentMethod);
+exports.retrieveAllCustomers = async (req, res) => {
```

Comment:
I don't think we'll need a route to retrieve all of the customers? :)

Reply:
I just added this in for testing with Postman I'll remove it when the PR is fully functional :)